lib/tasks/contributor_stats.rake

```diff
@@ -0,0 +1,10 @@
+namespace :fromthepage do
+  desc "transcription stats from the previous 24 hours"
+  task :contributor_stats, [:collection_id] => :environment do |t,args|
```

Comment:
It's very easy to set it to take one email; I think the "or more" isn't bad, but I'm investigating.